pkg/controller/nodekeeper/nodekeeper_controller.go

```diff
@@ -0,0 +1,225 @@
+package nodekeeper
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	"k8s.io/apimachinery/pkg/types"
+
+	"github.com/go-logr/logr"
+	configv1 "github.com/openshift/api/config/v1"
+	machineconfigapi "github.com/openshift/machine-config-operator/pkg/apis/machineconfiguration.openshift.io/v1"
+	upgradev1alpha1 "github.com/openshift/managed-upgrade-operator/pkg/apis/upgrade/v1alpha1"
+	"github.com/openshift/managed-upgrade-operator/pkg/metrics"
+	"github.com/openshift/managed-upgrade-operator/pkg/osd_cluster_upgrader"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
```

Comment:
We're kind-of establishing a convention of grouping our module imports into:
- standard library
- external modules
- internal modules
(Each group separated by a empty line) Take a look at the other controller for an example there. It's purely stylistic.

Reply:
This a project choice? The above is just from fmt.

Reply:
It's a project choice, but a broader style guide convention. Haoran impressed it upon me in a previous PR. :)  https://github.com/golang/go/wiki/CodeReviewComments has a section on it.

Reply:
resolved: 18a3825c33a3c99da7b8c7b5dd26992128533a46